pubnub-kotlin/pubnub-kotlin-api/src/jsMain/kotlin/Pubnub.d.kt

```diff
@@ -420,6 +422,7 @@ open external class PubNub(config: Any /* UUID | UserId */) {
         var message: Any
         var file: `T$5`
         var userMetadata: Any?
+        var customMessageType: String?
```

Comment:
Do we need this? Is there customMessageType related to MessageActionEvent?

Reply:
this is in FileEvent